yarn.lock

```diff
@@ -6470,6 +6470,11 @@ lodash@^4.13.1, lodash@^4.15.0, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.4
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.11.tgz#b39ea6229ef607ecd89e2c8df12536891cac9b8d"
```

Comment:
There will be another duplicate lodash 4.17.11 inside `node_modules`. Could you run
```
yarn-deduplicate --package lodash ./yarn.lock
```
[yarn-deduplicate](https://github.com/atlassian/yarn-deduplicate#limit-packages-to-deduplicate-yarnlock) is a third party tool helps keep dependency flat.

Reply:
Thanks! I ran it and amended it!